.github/workflows/test.yml

```diff
@@ -16,13 +16,21 @@ jobs:
         with:
           python-version: ${{ env.PYTHON_VERSION }}
           
-      - name: Install RgxLog
-        run: |
-          python -m pip install --upgrade pip
-          pip install -e ./spanner_workbench
-          
       - name: Use nbdev CI Workflow
         uses: fastai/workflows/nbdev-ci@master
         with:
           # The python version that nbdev ci must run on must match the python version used 
           version: ${{ env.PYTHON_VERSION }}
+          skip_test: true
+          
+      - name: Run Tests
+        run: |
+          # Capture both stdout and stderr to a temporary file
+          python nbdev_test.py > test_output.txt
+          cat test_output.txt  # Print the test output for debugging purposes
+
+          if grep -q "nbdev Tests Failed On The Following Notebooks:" test_output.txt; then
+            echo "nbdev Tests Failed"
+            exit 1
+          fi
```

Comment:
write the test_output.txt to a tmpfile in /tmp/ directory so the script can be run locally without polluting the staging area.